build.gradle

```diff
@@ -144,6 +141,15 @@ task generateInfo(type: DefaultTask) {doLast{
 	new FileOutputStream(buildInfoFile).withCloseable({ properties.store(it, "Build info") })
 }}
 
+// copy jar from the standard Gradle output (build/libs) to target/
+// This allows JitPack to find the output jar and archive it
+task copyFinalJarToTarget(type: Copy) {
```

Comment:
copyFinalJarToLibs is alot more clear. Also you probably don't need those comments.

Reply:
? It copies from `build/libs` to `target/` and not in reverse order.

I would keep line 145, because it states why we introduced this change.

Reply:
@games647 Yeah I saw, edited before I read your answer 😄